docs/src/transforms.md

```diff
@@ -528,6 +534,8 @@ The response from the down-chain transform is returned back up-chain but various
     #         filter: Read
     #     - NullSink
 
+    # Optional port to for the HTTP chain switcher API to listen on
+    # switch_port: 1234
```

Comment:
Maybe give this a generic name like "http_api_port" to allow it to be extended with other configurations in the future?